public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -198,8 +208,20 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
             onIntervalChanged={onChangeRefreshInterval}
             value={refreshInterval}
             isLoading={loading}
-            text={showSmallTimePicker ? undefined : loading ? 'Cancel' : 'Run query'}
-            tooltip={showSmallTimePicker ? (loading ? 'Cancel' : 'Run query') : undefined}
+            text={
+              showSmallTimePicker
+                ? undefined
+                : loading
+                ? t('grafana-ui.refresh-picker.cancel', 'Cancel')
```

Comment:
It doesn't need to be prefixed with `grafana-ui`. For generic components, we can use 2-level IDs, e.g. `refresh-picker.cancel`. 

On the other hand, it's a generic component used inside Explore, and Explore provides the translation, so we could also prefix it with `explore.`. However, we may want to keep consistent names across Dashboards/Explore so maybe it should be turned into a separate component in long run? @grafana/grafana-frontend-platform any guidelines on how to tackle such use cases? I think we can go with a 2-level ID here but wanted to double-check with you.

Reply:
I wouldn't say this is exactly a generic component because this text it is only used in Explore. In dashboards the tooltip is already translated with `tooltip={t('dashboard.toolbar.refresh', 'Refresh dashboard')}` so something like `explore.toolbar.refresh-picker-cancel`? To keep in line with the other way you are translating things in this file.

Also, we can't use the same key in multiple places, as I see we are using here for `tooltip` and `text`. We should abstract this into a constant that we can use in both places if they are the same anyway like:
```
const refreshPickerLabel = loading
                  ? t('explore.toolbar.refresh-picker-cancel', 'Cancel')
                  : t('explore.toolbar.refresh-picker-run', 'Run query')
```

and then use this in both `text` and `tooltip`?

Reply:
Having `explore.toolbar.refresh-picker-cancel` / `explore.toolbar.refresh-picker-run` + separate const makes sense to me 👍

Reply:
Thank you for the feedback. I have implemented it.